### PR TITLE
feat(components/colorpicker): adjust interface on smaller screens

### PR DESCRIPTION
**Pull request by johnhwhite**
[AB#2760092](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2760092)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1905/storybook/)

Component Storybooks:

- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1905/storybooks/colorpicker-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1905/storybooks/pages-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1905/storybooks/text-editor-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1905/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1905/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1905/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1905/playground/)

File: libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts [hunk 1]
```diff
@@ -546,8 +546,9 @@ export class SkyColorpickerComponent implements OnInit, OnDestroy {
         autoFitContext: SkyAffixAutoFitContext.Viewport,
         enableAutoFit: true,
         horizontalAlignment: 'left',
+        verticalAlignment: 'bottom',
         isSticky: true,
-        placement: 'below',
+        placement: 'right',
```
Comment (Blackbaud-SteveBrush) [inline libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts]: Can we avoid changing the expected placement behavior and only apply the CSS fix?

File: apps/integration/src/app/integrations/modal-colorpicker/modal-colorpicker.component.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { SkyModalService } from '@skyux/modals';
+
+import { ModalColorpickerModalComponent } from './modal-colorpicker-modal.component';
+
+@Component({
+  selector: 'app-modal-colorpicker',
+  templateUrl: './modal-colorpicker.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
```
Comment (Blackbaud-SteveBrush) [inline apps/integration/src/app/integrations/modal-colorpicker/modal-colorpicker.component.ts:9]: Standalone?

File: apps/integration/src/app/integrations/modal-colorpicker/modal-colorpicker.module.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import { ModalColorpickerRoutingModule } from './modal-colorpicker-routing.module';
+import { ModalColorpickerComponent } from './modal-colorpicker.component';
+
+@NgModule({
+  declarations: [ModalColorpickerComponent],
+  imports: [CommonModule, ModalColorpickerRoutingModule],
+})
+export class ModalColorpickerModule {
+  public static routes = ModalColorpickerRoutingModule.routes;
```
Comment (Blackbaud-SteveBrush) [inline apps/integration/src/app/integrations/modal-colorpicker/modal-colorpicker.module.ts:9]: ☝️ Nearly all of the public properties and methods (on the new files in this PR) can use `protected` instead.

File: libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts (hunk 1 above)
Comment (johnhwhite) [reply libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts]: I reverted the `placement` change, but I found the `verticalAlignment` setting important to get affixer to land on an acceptable position.

Comment (johnhwhite) [reply libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker.component.ts]: Bumped the risk to 2 since it may impact expected behavior.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**